cfme/fixtures/provider.py

```diff
@@ -382,7 +382,7 @@ def big_template_modscope(provider):
     return _get_template(provider, 'big_template')
 
 
-@pytest.fixture(scope="function")
+@pytest.fixture(scope="module")
```

Comment:
Note: To speed up automation; we need this. I don't want due to `provisioning` lower scope limitation, we can not create long setup fixture with module scope.